Lib/fontTools/cffLib/__init__.py

```diff
@@ -28,8 +39,37 @@
 
 
 class CFFFontSet(object):
+	"""A CFF font "file" can contain more than one font, although this is
+	extremely rare (and fontTools can't handle such files anyway...).
```

Comment:
It is been a while since I looked into this, but I think FonTools can handle “bare” CFF files with multiple fonts, but probably not in the CFF table since this is not allowed by the [spec](https://docs.microsoft.com/en-us/typography/opentype/spec/cff)